javatests/arcs/showcase/references/Arcs.kt

```diff
@@ -3,27 +3,68 @@
 package arcs.showcase.references
 
 import android.content.Context
-import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LifecycleRegistry
-import arcs.android.sdk.host.AndroidHost
 import arcs.core.allocator.Allocator
-import arcs.core.common.toArcId
+import arcs.core.allocator.Arc
+import arcs.core.common.ArcId
+import arcs.core.data.Plan
+import arcs.core.host.AbstractArcHost
 import arcs.core.host.EntityHandleManager
-import arcs.core.host.SchedulerProvider
+import arcs.core.host.ParticleState
 import arcs.core.host.toRegistration
-import arcs.core.storage.StoreManager
+import arcs.core.host.SchedulerProvider
+import arcs.core.storage.ActivationFactory
 import arcs.jvm.host.ExplicitHostRegistry
 import arcs.jvm.host.JvmSchedulerProvider
 import arcs.jvm.util.JvmTime
+import arcs.sdk.Particle
 import arcs.sdk.android.storage.ServiceStoreFactory
 import arcs.sdk.android.storage.service.ConnectionFactory
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
+import kotlinx.coroutines.withTimeout
 import kotlin.coroutines.EmptyCoroutineContext
 
+/** Container for WriteRecipe specific things */
+@ExperimentalCoroutinesApi
+class ArcsStorage(private val arcs: Arcs) {
+    private inline fun <T> run(crossinline block: suspend () -> T) = runBlocking {
+        withTimeout(15000) {
```

Comment:
If this is just to get more accurate test error info, can we add a comment to make this clear? It wasn't obvious to me when I was reading tests recently. We should also perhaps standardize on a constant here and put it in Archron test utils somewhere so it's easy to change everywhere if needed and easy to see where we do it.